tutorials/roofit/rf401_importttreethx.py

```diff
@@ -68,26 +68,10 @@ def makeTTree():
 
 # Create a binned dataset that imports contents of all ROOT.TH1 mapped by
 # index category c
-dh = ROOT.RooDataHist(
-    "dh",
-    "dh",
-    ROOT.RooArgList(x),
-    ROOT.RooFit.Import("SampleA", hh_1),
-    ROOT.RooFit.Import("SampleB", hh_2),
-    Import=("SampleC", hh_3),
-    Index=c,
-)
+dh = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), c, {"SampleA": hh_1, "SampleB": hh_2, "SampleC": hh_3})
 dh.Print()
 
-# Alternative constructor form for importing multiple histograms
-ROOT.gInterpreter.GenerateDictionary("std::pair<std::string, TH1*>", "map;string;TH1.h")
-ROOT.gInterpreter.GenerateDictionary("std::map<std::string, TH1*>", "map;string;TH1.h")
-hmap = ROOT.std.map("string, TH1*")()
-hmap.keepalive = list()
-hmap.insert(hmap.cbegin(), ROOT.std.pair("const std::string,TH1*")("SampleA", hh_1))
-hmap.insert(hmap.cbegin(), ROOT.std.pair("const std::string,TH1*")("SampleB", hh_2))
-hmap.insert(hmap.cbegin(), ROOT.std.pair("const std::string,TH1*")("SampleC", hh_3))
-dh2 = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), c, hmap)
+dh2 = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), c, {"SampleA": hh_1, "SampleB": hh_2, "SampleC": hh_3})
```

Comment:
```suggestion
dh2 = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), Index=c, Import={"SampleA": hh_1, "SampleB": hh_2, "SampleC": hh_3})
```

I think it's better to use here the constructor with the command arguments. With the additional keywords, it's easier to understand what the code does.

Reply:
Sir, as I had mentioned on mattermost previously, this leads to an error

Reply:
Right, I have suggested a solution to that error at the end of the PR thread. I hope it will work like that!